Sources/System/FilePath/FilePath.swift

```diff
@@ -62,6 +62,16 @@ public struct FilePath {
 
 // @available(macOS 10.16, iOS 14.0, watchOS 7.0, tvOS 14.0, *)
 extension FilePath {
+  
+  /// Returns the current working directory of this process
+  public static func getCurrentWorkingDirectory() throws -> FilePath {
+    guard let cwd = system_getcwd(nil, 0) else {
+      throw Errno.current
+    }
+    defer { system_free(cwd) }
+    return FilePath(platformString: cwd)
+  }
+  
```

Comment:
Way back when I was thinking this would be hosted on some kind of current process type, but I think this approach makes more sense. It's also available for typename elision IIUC:

```swift
func foo(_: FilePath)
let x = foo(.getCurrentWorkingDirectory())
```

I feel like making this a function rather than a throwing var makes sense (ignoring tools version concerns) as it will make a syscall.